lib/nacelle-pushowl-plugin.js

```diff
@@ -113,8 +150,15 @@ export default function (ctx) {
       payload.eventType === 'PRODUCT_VIEW' &&
       location.pathname.startsWith('/products/')
     ) {
-      handleProductView(payload.product)
+      const currentVariant =
+        state[`product/${payload.product.handle}`]?.selectedVariant
+
+      handleProductView(payload.product, currentVariant)
+      setFlyoutWidget(true)
       return
+    } else {
+      // Hide the flyout widget
+      setFlyoutWidget(false)
     }
 
     /**
```

Comment:
Can't see any semi-colons in this file. Is prettier configured to remove them?

Reply:
Yes, the current prettier config in this repo is 

```
{
  "semi": false,
  "singleQuote": true,
  "trailingComma": "none"
}
```